### ISSUE TITLE
How to label my own data?

### ISSUE THREAD

**Issue by 13052** [eragonruan/text-detection-ctpn#209]
Label text line or proposal box, or could I label text line then convert to proposal format?
Thank you in advance.

**Comment by eragonruan** [comment]
@13052 just label in ground truth in line level, and use the scripts to convert it

**Comment by hsonetta** [comment]
Hi , How did you annotate your data while preparing the training dataset? Are we supposed to use some software to get the label cordinates from images ?
